### DISCUSSION TITLE
我云服务器安装了1panel面板

### DISCUSSION THREAD

**Discussion by anketong** [kkkgo/PaoPaoDNS#215] [问答和帮助]
我云服务器安装了1panel面板，通过1panel面板直接安装adguardhome ，后通过在1panel shh安装了paopaodns,   adguardhome上游设置paopaodns(ip+54)显示无法连接  但是设置为adguardhome Bootstrap DNS 服务器解析上游的加密 又可以用。 我该如何解决?
ps我该放行的端口均已放行。
![1](https://github.com/user-attachments/assets/fa905692-0250-4f18-ac33-75f2ac0dab68)
![2](https://github.com/user-attachments/assets/229d7c78-fb21-422b-9118-ae552c84ec89)
![3](https://github.com/user-attachments/assets/d6ae3388-c18a-4b64-acd9-25e167b69981)

**Comment by kkkgo** [comment]
54端口是啥，没有这个定义

**Reply by anketong** [reply]
哥 我改会玩53了也不行 我在这里纠结了一天了 什么都查询了  我都正常 就是adguardhome上游填不了。麻烦你给我解答一下嘛 谢谢

**Reply by anketong** [reply]
能否给个具体联系消息，我有偿，我在这里干看。谢谢你了

**Reply by kkkgo** [reply]
如果按你这个界面来看，你想映射为54端口就应该在  服务器框 填54，在 容器框 填53，这就是54映射到53端口的意思，两个都填54就不知道在映射啥了。

**Reply by anketong** [reply]
可以了. ipv6没开
